src/encoder/encodervorbis.cpp

```diff
@@ -37,39 +37,15 @@ A lot of stuff has been stolen from:
 
 EncoderVorbis::EncoderVorbis(EncoderCallback* pCallback)
         : m_bStreamInitialized(false),
+          m_vblock({}),
```

Comment:
Yes, that's right. It looks like we should move it to the constructor and refactor the calling updateFromPreferences() a bit.  
But since the current solution already initializes the members of question, it is only a beatify issue. Is it worth the work? 
In any case, we should not introduce a gcc 4.8 compiler warning.